src/kw_time_and_date.sh

```diff
@@ -86,6 +86,8 @@ function days_in_the_month()
 {
   local month_number="$1"
   local year="$2"
+  local days=31
+  local short=(4 6 9 11)
```

Comment:
It is not clear to me why you created this array with these numbers.

Reply:
These are the months that have 30 days, the rest have 31. Except for February.